sonar-python-plugin/src/test/java/org/sonar/plugins/python/indexer/SonarQubePythonIndexerTest.java

```diff
@@ -250,14 +255,40 @@ public void test_no_file_modified_missing_descriptors() {
       .contains("2/2 source files have been analyzed");
   }
 
```

Comment:
As discussed, we can have a test asserting that a change in `sonar.python.version` invalidates the cache